whatlies/embedding.py

```diff
@@ -212,29 +220,32 @@ def plot(
         bar.plot(kind="arrow", annot=True)
         ```
         """
-        if len(self.vector) == 2:
-            handle_2d_plot(
-                self,
-                kind=kind,
-                color=color,
-                show_operations=show_ops,
-                xlabel=x_axis,
-                ylabel=y_axis,
-                annot=annot,
-                axis_option=axis_option,
-            )
-            return self
-        x_val = self > x_axis
-        y_val = self > y_axis
-        intermediate = Embedding(name=self.name, vector=[x_val, y_val], orig=self.orig)
+        x_val, x_lab = self._get_plot_axis_value_and_label(x_axis, dir="x")
+        y_val, y_lab = self._get_plot_axis_value_and_label(y_axis, dir="y")
+        x_label = x_lab if x_label is None else x_label
+        y_label = y_lab if y_label is None else y_label
+        emb_plot = Embedding(name=self.name, vector=[x_val, y_val], orig=self.orig)
         handle_2d_plot(
-            intermediate,
+            emb_plot,
             kind=kind,
             color=color,
-            xlabel=x_axis.name,
-            ylabel=y_axis.name,
+            xlabel=x_label,
+            ylabel=y_label,
+            title=title,
             show_operations=show_ops,
             annot=annot,
             axis_option=axis_option,
         )
         return self
+
+    def _get_plot_axis_value_and_label(self, axis, dir):
```

Comment:
You can still add a docstring to this method. Because there is an underscore at the start of the name, `mkdocs` will automatically understand that it should not appear in the API docs.

Reply:
I don't think the docstring needs to be as explicit as a one for a user-facing method. But it can be useful for internal use if there's a mention why the function was created.